src/Compilers/CSharp/Portable/CSharpResources.resx

```diff
@@ -4604,6 +4607,8 @@ To remove the warning, you can use /reference instead (set the Embed Interop Typ
                               diagnostics.
 -errorlog:&lt;file&gt;              Specify a file to log all compiler and analyzer
                               diagnostics.
+-sarifversion:[1|2]           Specify the version of the SARIF format in the
```

Comment:
implementation doesn't match this.  Other values are allowed (like `2.1.0`).  I would either have the implementation match this, or update this to actually match impl.